apps/frontend/src/app/PageTeam/CharacterDisplay/Tabs/TabOptimize/index.tsx

```diff
@@ -1155,32 +1141,34 @@ function BuildItemWrapper({
   allowLocationsState: AllowLocationsState
 }) {
   const { t } = useTranslation('page_character_optimize')
-
+  const extraButtonsLeft = useMemo(() => {
```

Comment:
How much time did this take to render? On which interactions does this save a render?

Reply:
I believe the `onClick={()=>...}` of the button is triggering renders inside the `BuildDisplayItem`. This memorization prevents that from happening. (an argument can be made to use `useCallback` for that function, but I think this also prevents the `<></>` from triggering updates as well)